docs/concepts/05_credentials/05_verification.md

```diff
@@ -3,68 +3,97 @@ id: verification
 
-- The credential is valid (i.e., not revoked by its Attester)
-- The attributes in the credential actually refer to the Claimer 
-- The credential contains information that is relevant for the Verifier in this use case
+-   The credential is valid (i.e., not revoked by its Attester)
+-   The attributes in the credential actually refer to the Claimer
+-   The credential contains information relevant for the Verifier for this use case
 
-## Requesting a Credential from a Claimer
+## Requesting a credential from a Claimer
 
 The Verifier may request a credential from a Claimer, providing the following data:
-- **CTypes**: which CTypes the Verifier can work with for the use case. They can provide multiple options, which have to be regarded as alternatives.
-- **TrustedAttesters**: which Attesters are considered trusted for each specified CType.
-- **RequiredProperties**: which properties for each specified CType must at least be revealed for the presentation to be considered sufficient. More on "selective disclosure" later.
-- **Challenge**: a nonce, which the Verifier can make use of to ensure that the presentation generated by the Claimer is fresh and is not being replayed by some other older interactions. More on that later.
 
-## Presenting a Credential with Selective Disclosure
+-   `cTypeHash`: Which CType hashes the Verifier can work with for the use case. They can provide multiple options, to regard as alternatives.
+-   `trustedAttesters`: Which Attesters to consider trusted for each specified CType.
+-   `requiredProperties`: Which properties for each specified CType must at least be revealed for the Verifier to consider the presentation sufficient.
 
-Given the `RequiredProperties` specified by the Verifier, the Claimer can decide how much of the information they wish to reveal before they generate the presentation and send it to the Verifier.
-If supported by the Verifier, they can choose to hide attributes and thus only disclose a subset of the original claim data.
+    :::info
+
+    [Read more on selective disclosure](#presenting-a-credential-with-selective-disclosure).
+
+    :::
+<!-- TODO: Only example I can find that shows this has it outside of the CTypes -->
+-   `challenge`: A nonce, which the Verifier can use to ensure that the presentation generated by the Claimer is fresh and not replayed by some other older interactions.
+
+### Example
 
-For example, if verifying a driving license only requires the verification of the driver's name and picture, a Claimer can decide to hide additional information such as age and place of residence.
-This increases the privacy of the Claimer since they only need to show attributes that are required in the specific context.
+```json
+  "requirements": [{
+    "cTypeHash":"0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac",
+    "trustedAttesters":["did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY"],
+    "requiredProperties": ["Email"]
+  }],
+```
```

Comment:
The challenge should be there as well. 

This is an example from the [web3-login-demo](https://github.com/KILTprotocol/web3-login-demo), that uses the kilt sdk to wrap a `Kilt.IRequestCredential` in a `Kilt.IMessage`

**the message with the Credential-Request before encryption:** 
```suggestion
 {
   "body": {
      "content": {
         "cTypes": [
            {
               "cTypeHash": "0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac",
               "trustedAttesters": [
                  "did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY"
               ],
               "requiredProperties": [
                  "Email"
               ]
            },
            {
               "cTypeHash": "0x47d04c42bdf7fdd3fc5a194bcaa367b2f4766a6b16ae3df628927656d818f420",
               "trustedAttesters": [
                  "did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY"
               ],
               "requiredProperties": [
                  "Twitter"
               ]
            }
         ],
         "challenge": "0x5a1a17eca9ddf6b4d14dffb2abd1411fe9235927975a246b3963db86dfb7de5f"
      },
      "type": "request-credential"
   }
}
```

If it is too big, you could use only the scope inside of `content`. This means unwrapping it from the message. 

Or just one of the cTypes.